.github/workflows/cd-helm-workflow.yml

```diff
@@ -22,8 +22,14 @@ on:
     - 'aio/scripts/helm-release-chart.sh'
     - '.github/workflows/cd-helm-workflow.yml'
 
+permissions:
+  contents: read
```

Comment:
Is it necessary since we have only one job and you change the permissions below?

Reply:
Yes, the contents: read is not strictly needed here. Setting that at the workflow level helps if a new job is added to the workflow. In that case, the new job will have contents: read, which is a secure-by-default setting.